Bottom: 9a31f45b94fda88d1468a34b541fcc867a5b8c9f
Top:    60d3d5c87545a77e2d15b06bf8ad6c3b88fb6b00
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-02-10 13:56:23 +0000

Refresh of add-raw-polygons-support-for-t

---

diff --git a/src/global.h b/src/global.h
index 88e64e7..93aef49 100644
--- a/src/global.h
+++ b/src/global.h
@@ -245,7 +245,7 @@ struct polygon_st			/* holds information about a polygon */
   POLYAREA *NoHoles;		/* the polygon broken into hole-less regions */
   int NoHolesValid;		/* Is the NoHoles polygon up to date? */
   PointTypePtr Points;		/* data */
-} PolygonType, *PolygonTypePtr;
+};
 
 typedef struct			/* holds information about arcs */
 {
